ms_active_directory/core/ad_objects.py

```diff
@@ -170,6 +170,14 @@ def __repr__(self):
     def __str__(self):
         return self.__repr__()
 
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
```

Comment:
computers are also users, so I'll probably override this in `ADComputer` and `ADUser`